src/migrations/1633155784833-application-sort.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class applicationSort1633155784833 implements MigrationInterface {
```

Comment:
The migration must also take care of the existing data.

Reply:
The migration does take care of it:
![image](https://user-images.githubusercontent.com/30729240/136012091-0143a897-1ab7-475f-b43e-045fe2c04346.png)
The field gets a default value of zero, which is fine. It means existing applications are still not sorted, but that is something we just live with. Any new applications wil be properly sorted.